roles/custom/matrix-bridge-hookshot/templates/config.yml.j2

```diff
@@ -107,6 +107,14 @@ metrics:
   # (Optional) Prometheus metrics support
   #
   enabled: {{ matrix_hookshot_metrics_enabled | to_json }}
+{% if matrix_hookshot_experimental_encryption_enabled %}
+queue:
+  monolithic: true
+  port: 6379
+  host: matrix-redis
+experimentalEncryption:
+  storagePath: /data/encryption
```

Comment:
This should be out of the `if` that deals with the `queue` stuff, since one may wish to enable the queue without enabling experimental encryption.

Experimental encryption does probably require the `queue` configuration, but I suppose Hookshot checks for this by itself. You may add a check in `validate_config.yml` if you're feeling like it though.

`experimentalEncryption` settings should be dependant on `{% if matrix_hookshot_experimental_encryption_enabled %}`.